components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java

```diff
@@ -902,14 +916,17 @@ private void validateOrganizationStatusUpdate(String value, String organizationI
         }
     }
 
-    private List<ExpressionNode> getExpressionNodes(String filter, String after, String before)
+    private List<ExpressionNode> getExpressionNodes(String filter, String after, String before,
+                                                    boolean isFilteringMetaAttributes)
```

Comment:
This boolean property can be misunderstood as whether `filter` param has meta attributes as well, even thiugh the intention is to imply that if we are filtering attribute objects not the org objects.
However, to make `getExpressionNodes` function more generic for any usecase let's consider the result sorting order as mentioned in https://github.com/wso2/identity-organization-management-core/pull/144/files#r1683792495

Reply:
The boolean value is now removed and replaced by the string paginationSortOrder, which can take the values ASC or DESC.